bullet_train-super_scaffolding/lib/scaffolding/script.rb

```diff
@@ -7,6 +7,27 @@
 
 require_relative "../bullet_train/terminal_commands"
 
+FIELD_PARTIALS = {
+  boolean: "boolean",
+  buttons: "string",
+  cloudinary_image: "string",
+  color_picker: "string",
+  date_and_time_field: "datetime",
+  date_field: "date",
+  email_field: "string",
+  emoji_field: "string",
+  file_field: "attachment",
+  image: "attachment",
```

Comment:
Should this be `active_storage_image`? Or should we remove `cloudinary_image` above and then have `image` be a special case where we don't do a direct lookup, but instead check whether Cloudinary is active?